src/Symfony/Bundle/FrameworkBundle/Resources/config/error.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+use Symfony\Bundle\FrameworkBundle\Error\PagesDumper;
+use Symfony\WebpackEncoreBundle\Asset\EntrypointLookupInterface;
+
+return static function (ContainerConfigurator $container) {
+    $container->services()
+        ->set('error.pages_dumper', PagesDumper::class)
+        ->args([
+            param('kernel.cache_dir'),
```

Comment:
The destination folder could be defined in the semantic configuration. Maybe with `%kernel.cache_dir%/error_pages/` as default value

Reply:
Probably build_dir then ?